qa/tasks/mgr/dashboard/test_config_options.py

```diff
@@ -0,0 +1,54 @@
+import time
+
+from .helper import DashboardTestCase, authenticate
+
+
+class ConfigOptionsTest(DashboardTestCase):
+
+    @authenticate
+    def test_get_db_config_options(self):
+        data = self._get('/api/config_options')
+        self.assertStatus(200)
+        self.assertIsInstance(data, list)
+
+    @authenticate
+    def test_get_daemon_runtime_config(self):
+        def _get_mon_allow_pool_delete_config():
+            data = self._get('/api/config_options?section=mon.a')
+            config_option = [c for c in data if c['name'] == 'mon_allow_pool_delete']
+            return config_option[0]['value']
+
+        self._ceph_cmd(['tell', 'mon.a', 'injectargs', '--no-mon-allow-pool-delete'])
+        result = self._wait_for_expected_get_result(_get_mon_allow_pool_delete_config, 'false')
+        self.assertEqual(result, 'false')
+
+        self._ceph_cmd(['tell', 'mon.a', 'injectargs', '--mon-allow-pool-delete'])
+        result = self._wait_for_expected_get_result(_get_mon_allow_pool_delete_config, 'true')
+        self.assertEqual(result, 'true')
+
+    @authenticate
+    def test_get_specific_db_config_option(self):
+        def _get_mon_allow_pool_delete_config():
+            data = self._get('/api/config_options/mon_allow_pool_delete')
+            config_option = [c for c in data if c['section'] == 'mon']
+            return config_option[0]['value']
+
+        self._ceph_cmd(['config', 'set', 'mon', 'mon_allow_pool_delete', 'true'])
+        result = self._wait_for_expected_get_result(_get_mon_allow_pool_delete_config, 'true')
+        self.assertEqual(result, 'true')
+
+        self._ceph_cmd(['config', 'set', 'mon', 'mon_allow_pool_delete', 'false'])
+        result = self._wait_for_expected_get_result(_get_mon_allow_pool_delete_config, 'false')
+        self.assertEqual(result, 'false')
+
+    def _wait_for_expected_get_result(self, get_func, expected_result, max_attempts=3,
+                                      sleep_time=10):
```

Comment:
could you please only specify the timeout or the number of attepts (e.g. 30 seconds or attempts) and then try again every second?